.pre-commit-config.yaml

```diff
@@ -0,0 +1,21 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.2.0
+    hooks:
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
+      - id: check-json
+      - id: check-merge-conflict
+  - repo: https://github.com/awslabs/git-secrets
+    rev: 80230afa8c8bdeac766a0fece36f95ffaa0be778
+    hooks:
+      - id: git-secrets
+        entry: "git-secrets --scan"
+        files: "."
+  - repo: https://github.com/aws-cloudformation/cfn-python-lint
+    rev: v0.37.0
+    hooks:
+      - id: cfn-python-lint
+        name: AWS CloudFormation Linter
+        files: \.(template)$
```

Comment:
IDK about you, but I don't use that suffix when I write CFn templates. This seems like a per-project detail.

Reply:
Agreed, not everybody uses it but our most prominent source of templates (<https://github.com/awslabs/aws-cloudformation-templates/>) does (mostly) and setting this to include `.json` and `.yaml` would catch files that aren't intended. Including this here could be a nice incentive to get people to start using `.template`.

Reply:
Yeah I agree. I personally use either `.template` or `.yaml` but it's not feasible to include `.json` and `.yaml` files as Steve says, and it's non-intrusive - if you don't have any `.template` files, the check is skipped